### PR TITLE
issue 470 market setting

### PR DESCRIPTION
**Pull request by thehapax**
#470 edit display for end user so it reads QUOTE/BASE instead of BASE/QUOTE

### CODE REVIEW

File: dexbot/strategies/base.py [hunk 1]
```diff
@@ -154,7 +154,7 @@ def configure(cls, return_base_config=True):
                           'BitShares account name for the bot to operate with',
                           ''),
             ConfigElement('market', 'string', 'USD:BTS', 'Market',
```
Comment (bitfag) [inline dexbot/strategies/base.py]: Default value USD:BTS should be changed too.

Comment (thehapax) [reply dexbot/strategies/base.py]: make it a separate issue please.

Comment (bitfag) [reply dexbot/strategies/base.py]: Why separate? This change is related to your change. You changed ordering in one place, so change the default too to make your change consistent.

Comment (thehapax) [reply dexbot/strategies/base.py]: what do you want USD:BTS to be changed to? BTS:USD or BTS/USD? Please be specific.

Comment (bitfag) [reply dexbot/strategies/base.py]: To BTS/USD

Comment (thehapax) [reply dexbot/strategies/base.py]: ok done.

File: dexbot/strategies/base.py [hunk 2]
```diff
@@ -153,9 +153,9 @@ def configure(cls, return_base_config=True):
             ConfigElement('account', 'string', '', 'Account',
                           'BitShares account name for the bot to operate with',
                           ''),
-            ConfigElement('market', 'string', 'USD:BTS', 'Market',
-                          'BitShares market to operate on, in the format ASSET:OTHERASSET, for example \"USD:BTS\"',
-                          r'[A-Z0-9\.]+[:\/][A-Z0-9\.]+'),
+            ConfigElement('market', 'string', 'USD/BTS', 'Market',
```
Comment (bitfag) [inline dexbot/strategies/base.py]: This assumes USD is QUOTE which is confusing, please change to BTS/USD.

Comment (thehapax) [reply dexbot/strategies/base.py]: done.

File: dexbot/strategies/base.py [hunk 3]
```diff
@@ -153,9 +153,9 @@ def configure(cls, return_base_config=True):
             ConfigElement('account', 'string', '', 'Account',
                           'BitShares account name for the bot to operate with',
                           ''),
-            ConfigElement('market', 'string', 'USD:BTS', 'Market',
-                          'BitShares market to operate on, in the format ASSET:OTHERASSET, for example \"USD:BTS\"',
-                          r'[A-Z0-9\.]+[:\/][A-Z0-9\.]+'),
+            ConfigElement('market', 'string', 'BTS/USD', 'Market',
+                          'BitShares market to operate on, in the format QUOTE/BASE, for example \"BTS/USD\"',
+                          r'[A-Z\.]+[:\/][A-Z\.]+'),
```
Comment (bitfag) [inline dexbot/strategies/base.py]: You lost changes introduced in 9a0fc56b417fac1c4dbde43a65bdb941709a2a58, please rebase your changes to latest `devel` branch

**Pull request merged**